vaadin-dialog-flow-parent/vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -37,16 +37,19 @@
 import com.vaadin.flow.component.dependency.JsModule;
 import com.vaadin.flow.dom.Element;
 import com.vaadin.flow.dom.ElementConstants;
+import com.vaadin.flow.dom.Style;
+import com.vaadin.flow.function.SerializableConsumer;
```

Comment:
Is this import used anywhere?

Reply:
`Style` is being used but `SerializableConsumer` is not.